mne/gui/__init__.py

```diff
@@ -15,6 +15,8 @@ def combine_kit_markers():
     The functionality in this GUI is also part of :func:`kit2fiff`.
     """
     _check_mayavi_version()
+    from ._backend import _check_backend
```

Comment:
We don't want GUI libraries to be imported unless the user actually wants them. I could move the two lazy imports to the start of each function though.